Geometry/RPCGeometryBuilder/src/RPCGeometryBuilder.cc

```diff
@@ -127,18 +122,15 @@ RPCGeometry* RPCGeometryBuilderFromDDD::buildGeometry(DDFilteredView& fview,
       bounds = new RectangularPlaneBounds(width, length, thickness);
       const std::vector<float> pars = {width, length, float(numbOfStrips.doubles()[0])};
 
-      if (!theComp11Flag) {
-        if (tran.z() > -1500.) {
-          Basic3DVector<float> newX(-1., 0., 0.);
-          Basic3DVector<float> newY(0., -1., 0.);
-          Basic3DVector<float> newZ(0., 0., 1.);
-          rot.rotateAxes(newX, newY, newZ);
-        }
+      if (tran.z() > -1500.) {
```

Comment:
@slomeo - this could be causing the problem. You should assume that the code is compatible with C++11.

Reply:
@ianna the comp11 variable is not present anymore, what do you suggest to write?

Reply:
please, remove the block